data/categories.txt

```diff
@@ -12464,3 +12499,818 @@ uk:Місце проведення подій
 vi:Địa điểm tổ chức sự kiện
 zh-Hans:活动场所
 zh-Hant:活動場所
+
+shop-chocolate|@food
+en:Chocolate Shop
+de:Schokoladengeschäft
+es:Tienda de chocolate
+fr:Chocolatier
+it:Choccolateria
+ja:チョコレート屋
+nl:Chocolaterie
+pl:Sklep z czekoladą
+pt:Loja de chocolates
+pt-BR:Loja de chocolates
+
+shop-coffee|@food
+en:Cofee store
+de:Kaffeegeschäft
+fr:Boutique de cafés
+it:Negozio di caffè
+ja:コーヒー屋
+nl:Koffiewinkel
+pl:Sklep z kawą
+pt:Loja de café
+pt-BR:Loja de café
+
+shop-fabric
+en:Fabric shop
+de:Textilgeschäft
+fr:Magasin de tissus
+it:Merceria
+nl:Stoffenwinkel
+pt:Loja de tecidos
+pt-BR:Loja de aviamentos
+
+shop-money_lender
```

Comment:
We already have all translations in types_strings.txt, they at least should be copy-pasted here.

Reply:
Then they should be fixed everywhere )

Reply:
What do you mean  _everywhere_? This translations are equal with categories.